includes/discovery/mempools/fortimail.inc.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+echo 'FORTIMAIL-MEMORY-POOL: ';
+// fmlSysMemCapacity: Total physical memory (RAM) installed (KB)
+// Conversion from kilobyte (base 2) to byte (base 10): Precision = 1024
+discover_mempool($valid_mempool, $device, 0, 'fortimail', 'Physical Memory', '1024', null, null);
```

Comment:
You need to do the `snmp_get()` before this otherwise it will discovery a mempool for all fortimail devices (which may be genuinely possible however you shouldn't do this).

Reply:
I've just tried to generate the test data but I'm facing the following issue:
SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) (SQL: select * from `devices` where `hostname` = 127.1.6.1 limit 1)

Reply:
as i remember some of the things has to be run in user mode and not as root
Maybe this helps:
https://docs.librenms.org/Developing/os/Test-Units/

Reply:
I've managed to get the script running. But even when i start the script from within my development directory, it only scans the production directory for data files: 
- Script started:  /opt/librenmsdev/scripts/save-test-data.php -o fortimail
- Snippet of the output:
  Scanning "/usr/snmpsim/data" directory for  *.snmpwalk, *.MVC, *.sapwalk, *.snmprec, *.dump data files...
  Directory "/usr/snmpsim/data" does not exist

Reply:
I was finally able to generate the requested test data.